api/src/services/ssi-service.ts

```diff
@@ -138,7 +140,8 @@ export class SsiService {
 	}
 
 	async publishSignedDoc(newDoc: IdentityDocumentJson): Promise<string> {
-		const txHash = await Identity.publish(newDoc, this.config);
+		const client = this.getIdentityClient();
+		const txHash = await client.publishDocument(newDoc);
```

Comment:
looks like the new version introduced some breaking changes

Reply:
yes there were some breaking changes but easily to handle